deliver/lib/deliver/app_screenshot.rb

```diff
@@ -187,6 +196,10 @@ def self.device_messages
       # This list does not include iPad Pro 12.9-inch (3rd generation)
       # because it has same resoluation as IOS_IPAD_PRO and will clobber
       return {
+        ScreenSize::IOS_67_MESSAGES => [
+          [1290, 2796],
+          [2796, 1290]
```

Comment:
You can add a trailing comma for conventions.

```suggestion
          [2796, 1290],
```